### PR TITLE
Switch to new test package

### PR DESCRIPTION
**Pull request by alexweininger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -575,5 +575,11 @@
     },
     "extensionDependencies": [
         "ms-vscode.azure-account"
-    ]
+    ],
+    "__metadata": {
```
Comment (nturinski) [inline package.json:579]: Where did this come from??

File: package-lock.json [hunk 2]
```diff
@@ -13249,6 +13249,18 @@
                 "@microsoft/1ds-post-js": "^3.2.3"
             }
         },
+        "@vscode/test-electron": {
```
Comment (nturinski) [inline package-lock.json:13252]: Weird, it seems like the `package-lock.json` updated but I don't see these changes reflected in the `package.json` itself.

File: package.json (hunk 1 above)
Comment (alexweininger) [reply package.json:579]: Added automatically by the new test package.

File: package-lock.json (hunk 2 above)
Comment (alexweininger) [reply package-lock.json:13252]: Fixed. I slipped up when reverting the package.json changes that the new test package automatically makes (it annoyingly changes all the whitespace, etc).

File: package.json (hunk 1 above)
Comment (nturinski) [reply package.json:579]: It's not in the Functions repo [though](https://github.com/microsoft/vscode-azurefunctions/blob/main/package.json#L1117). Did we explicitly remove it?

Comment (bwateratmsft) [reply package.json:579]: Yeah, I'd prefer to remove it. It's puzzling why it's being added at all, I think we should raise it with the package authors.

Comment (alexweininger) [reply package.json]: I removed it, but according to https://github.com/microsoft/vscode/issues/148975#issuecomment-1124930941 it's intentionally added and "can be used later for other features like updating etc.,"

Comment (bwateratmsft) [reply package.json]: I think it makes sense for VSCode to update package.json in the installed extensions folder, but I don't think it should be doing it in source (like it currently is here).

**Review by nturinski — APPROVED** [review]

**Pull request merged**